README.md

```diff
@@ -27,6 +27,7 @@ Initialize your project using one of the following commands:
 - `npm init @shopify/app@latest` (installed by default with Node)
 - `pnpm create @shopify/create-app@latest`
 - `yarn create @shopify/app`
+- `bun create @shopify/app@latest`
```

Comment:
I think until we're confident you can do all the things you might want to do, via bun, we should make this clear that support is experimental. I gather we have more testing to do to reach that level of confidence